conf/core/ConfigOptions.asciidoc

```diff
@@ -1203,6 +1203,157 @@ Write out JSON in a more legible manner
 A list of tag keys for which the KeepTagsVisitor will retain on elements, while removing all
 other tags from elements.
 
+=== language.translation.comparison.pretranslated.tag.keys
+
+* Data Type: list
+* Default Value:
+** ``
+
+A list of tag keys known to have been previously translated to the desired target language.  The
+list ordering should correspond to that of language.translation.comparison.to.translate.tag.keys.
+Separate keys with a ';' and surround in quotes when passing in from the command line.
+
+=== language.translation.comparison.scorer
+
+* Data Type: string
+* Default Value: `hoot::LevenshteinDistance`
+
+StringDistance implementation used to score to English translated tag values
+
+=== language.translation.detected.language.overrides.specified.source.languages
+
+* Data Type: bool
+* Default Value: `false`
+
+If source languages are specified that do not match the detected language for piece of text during
+translation, setting this to try allows the detected language to override the specified source
```

Comment:
Should say, "... setting this to _true_ allows the detected ..."

Reply:
yes.  setting it to "try" wouldn't do too much :-)  thanks